gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -220,16 +222,6 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
         impl_->control_period_.seconds());
   }
 
-  // Initialize the emergency stop code.
```

Comment:
Isn't this supported anymore? I still see `e_stop_topic` on the demo URDF

Reply:
yes, it's supported